### ISSUE TITLE
Too tight distance to subscript

### ISSUE THREAD

**Issue by callegar** [stipub/stixfonts#214]
When using the font with luatex, in $V_Y$ the top left corner of the Y almost clashes onto the V.

**Comment by tiroj** [comment]
We’ll be thoroughly reviewing all such spacing in the v2.20 work, but if you are able to provide screenshots of the issue, that will be very helpful in determining whether some spacing problems are specific to individual math layout implementations.

**Comment by matteocoder** [comment]
@tiroj I agree that the space between a letter and its subscript is a bit too tight in STIX 2 Math compared to Latin Modern or even the older XITS.

```
% compiled with LuaHBTeX 1.13.2
\documentclass{minimal}
\usepackage{unicode-math}
% uncomment accordingly to produce each result
%\setmainfont{STIX Two Text}
%\setmathfont{STIX Two math}
%\setmainfont{XITS}
%\setmathfont{XITS Math}
\begin{document}
\noindent in text:\\
this is a $V_Y$ document\\
this is a $V_X$ document\\
this is a \(V_Y\) document\\
display:\\
this is a \[V_Y\] document
\end{document}
```
Results: STIX 2, Latin Modern, XITS
![stix2vy](https://user-images.githubusercontent.com/22897138/126870051-4bdd38a3-62b8-4a2e-8a7b-984ae0893dbe.JPG)
![lmvy](https://user-images.githubusercontent.com/22897138/126870055-ddb02fd6-2b1c-4af8-b661-47313d9557e9.JPG)
![xitsvy](https://user-images.githubusercontent.com/22897138/126870060-2d6c666a-3319-49a9-bd90-0b7a3556ded9.JPG)

**Comment by tiroj** [comment]
Thank you. The images are very helpful. We revised the italic correction setting on a lot of glyphs in v2.10, to make them more systematic, but now need to review and revise the downstream cut-in kerning settings.

**Comment by davidmjones** [comment]
Duplicate of #184
